onnx/utils.py

```diff
@@ -21,3 +24,138 @@ def polish_model(model):  # type: (ModelProto) -> ModelProto
     model = onnx.optimizer.optimize(model)
     onnx.checker.check_model(model)
     return model
+
+
+class Extractor:
+    def __init__(self, model):  # type: ignore
```

Comment:
it would be best if you add type annotations.

Reply:
All member functions have type annotations except two which are sort complex...